-1

Tested at: 47db651f030e95acade42adda7cecc0088f92980

CMSSW: CMSSW_11_2_PY3_X_2020-06-10-2300
SCRAM_ARCH: slc7_amd64_gcc820
You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-222956/6963/summary.html

I found follow errors while testing this PR

**Failed tests:  Build  HeaderConsistency**

* **Build**:

I found compilation error when building: 

<pre>/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/slc7_amd64_gcc820/external/pythia8/302/include/Pythia8/Weights.h:140:7: warning: base class 'class Pythia8::WeightsBase' has accessible non-virtual destructor [-Wnon-virtual-dtor]
 class WeightsLHEF : public WeightsBase {
       ^~~~~~~~~~~
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/slc7_amd64_gcc820/external/pythia8/302/include/Pythia8/Weights.h:140:7: warning: 'class Pythia8::WeightsLHEF' has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]
In file included from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_PY3_X_2020-06-10-2300/src/GeneratorInterface/Pythia8Interface/src/BiasedTauDecayer.cc:2:
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_PY3_X_2020-06-10-2300/src/GeneratorInterface/Pythia8Interface/interface/BiasedTauDecayer.h:15:20: error: 'Pythia8::Couplings' has not been declared
                    Pythia8::Couplings* couplingsPtr);
                    ^~~~~~~
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_PY3_X_2020-06-10-2300/src/GeneratorInterface/Pythia8Interface/src/BiasedTauDecayer.cc:12:89: error: 'Couplings' has not been declared
     Info* infoPtr, Settings* settingsPtr, ParticleData* particleDataPtr, Rndm* rndmPtr, Couplings* couplingsPtr) {
                                                                                         ^~~~~~~~~
</pre>

--- a/CondCore/AlignmentPlugins/test/BuildFile.xml
+++ b/CondCore/AlignmentPlugins/test/BuildFile.xml
@@ -3,7 +3,4 @@
 <use name="Alignment/CommonAlignment"/>
 <use name="CalibTracker/StandaloneTrackerTopology"/>
 <bin file="testTrackerAlignmentPayloadInspector.cpp" name="testTrackerAlignmentPayloadInspector">
- <ifrelease name="_PY3_">
-    <flags SETENV="PYTHONHOME=$(PYTHON_BASE)"/>
- </ifrelease>
 </bin>

--- a/CondCore/HLTPlugins/test/BuildFile.xml
+++ b/CondCore/HLTPlugins/test/BuildFile.xml
@@ -1,7 +1,4 @@
 <use name="CondCore/Utilities"/>
 <use name="FWCore/PluginManager"/>
 <bin file="testAlCaRecoTriggerBitsPayloadInspector.cpp" name="testAlCaRecoTriggerBitsPayloadInspector">
- <ifrelease name="_PY3_">
-    <flags SETENV="PYTHONHOME=$(PYTHON_BASE)"/>
- </ifrelease>
 </bin>

--- a/CondCore/RunInfoPlugins/test/BuildFile.xml
+++ b/CondCore/RunInfoPlugins/test/BuildFile.xml
@@ -1,7 +1,4 @@
 <use name="CondCore/Utilities"/>
 <use name="FWCore/PluginManager"/>
 <bin file="testRunInfoPayloadInspector.cpp" name="testRunInfoPayloadInspector">
- <ifrelease name="_PY3_">
-    <flags SETENV="PYTHONHOME=$(PYTHON_BASE)"/>
- </ifrelease>
 </bin>

--- a/CondCore/SiPixelPlugins/test/BuildFile.xml
+++ b/CondCore/SiPixelPlugins/test/BuildFile.xml
@@ -4,7 +4,4 @@
 <use name="CondFormats/SiPixelTransient"/>
 <use name="CalibTracker/SiPixelESProducers"/>
 <bin file="testSiPixelPayloadInspector.cpp" name="testPixelPayloadInspector">
- <ifrelease name="_PY3_">
-    <flags SETENV="PYTHONHOME=$(PYTHON_BASE)"/>
- </ifrelease>
 </bin>

--- a/CondCore/SiStripPlugins/test/BuildFile.xml
+++ b/CondCore/SiStripPlugins/test/BuildFile.xml
@@ -3,7 +3,4 @@
 <use name="FWCore/PluginManager"/>
 <use name="CalibTracker/StandaloneTrackerTopology"/>
 <bin file="testSiStripPayloadInspector.cpp" name="testSiStripPayloadInspector">
-  <ifrelease name="_PY3_">
-    <flags SETENV="PYTHONHOME=$(PYTHON_BASE)"/>
-  </ifrelease>
 </bin>

--- a/CondCore/Utilities/test/BuildFile.xml
+++ b/CondCore/Utilities/test/BuildFile.xml
@@ -10,7 +10,4 @@
 </bin>
 
 <bin file="testPngHistograms.cpp" name="testPngHistograms">
- <ifrelease name="_PY3_">
-    <flags SETENV="PYTHONHOME=$(PYTHON_BASE)"/>
- </ifrelease>
 </bin>

